### PR TITLE
i2260: Get new two-part metrics system working for barman in EC2

### PR DESCRIPTION
**Pull request by MartinEden**
https://vimc.myjetbrains.com/youtrack/issue/VIMC-2260

### CODE REVIEW

File: backup/aws/bin/run-barman.sh [hunk 1]
```diff
@@ -6,9 +6,10 @@ user_and_host="aws@$db_host"
 source ./db_passwords && rm ./db_passwords
 
 mkdir barman && cd barman
-git clone https://github.com/vimc/montagu-db
+git clone --recursive https://github.com/vimc/montagu-db
 cd montagu-db/backup
-git checkout master
+git checkout i2260  # Return to master
```
Comment (MartinEden) [inline backup/aws/bin/run-barman.sh]: This needs to be returned to master as part of the merge.

File: backup/start-metrics.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+set -ex
+
+cat <<EOF > config.json
+{
+  "labels": {
+    "db_name": "montagu"
+  },
+  "cache_volume": "barman_metrics",
+  "cache_filename": "metrics.json",
+  "port": 5000,
```
Comment (littledice) [inline backup/start-metrics.sh]: Port should be configurable/passed in as an option, unless the idea is to only use this script on EC2, where we're sure port 5000 is available?

File: backup/aws/bin/run-barman.sh [hunk 3]
```diff
@@ -33,7 +34,8 @@ autossh -M 20000 \
 # -u makes Python not buffer stdout, so we can monitor remotely
 # localhost is forwarded by SSH to the true host
 # For unknown reasons, this line refuses to break over multiple lines
-python3 -u ./barman-montagu setup --pull-image --image-source=vimc --no-clean-on-error --no-initial-backup --slot=barman_aws --volume_data=/mnt/data/barman_data --volume_logs=/mnt/data/barman_logs --volume_recover=/mnt/data/barman_recover localhost
+python3 -u ./barman-montagu setup --pull-image --image-source=vimc --no-clean-on-error --slot=barman_aws --volume_data=/mnt/data/barman_data --volume_logs=/mnt/data/barman_logs --volume_recover=/mnt/data/barman_recover localhost
+./start-metrics.sh
```
Comment (littledice) [inline backup/aws/bin/run-barman.sh]: Reorder these

File: backup/aws/barman_ssh_client.py [hunk 4]
```diff
@@ -54,12 +54,12 @@ def run_barman(self):
...
 
-        print("Installing machine metrics service...")
-        self._run_long_remote_cmd("./run-machine-metrics.sh")
-
         print("Running barman...")
         self._run_long_remote_cmd("./run-barman.sh")
 
+        print("Installing machine metrics service...")
+        self._run_long_remote_cmd("./run-machine-metrics.sh")
+
```
Comment (littledice) [inline backup/aws/barman_ssh_client.py]: Restore the previous order

**Pull request merged**